apps/music/js/ui/views/mode_manager.js

```diff
@@ -45,14 +45,14 @@ var ModeManager = {
   start: function(mode, callback) {
     this._modeStack = [];
     this._resetViews();
-    this._pushView(this.views[mode], false, function() {
+    this._pushView(mode, false, function() {
```

Comment:
@jimporter I think if we want to change the first argument  from to `view` to 'mode', then the the function name should probably change to `_pushMode`. I am fine with either approaches, just make sure the function name matches to the given argument.